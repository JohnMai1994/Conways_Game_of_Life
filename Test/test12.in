new 3
init
0 1 1 1 2 1
-1 -1
print
step
print
step
print
