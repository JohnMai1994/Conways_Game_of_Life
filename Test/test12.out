___
XXX
___
_X_
_X_
_X_
___
XXX
___
